CMakeLists.txt

```diff
@@ -2,13 +2,13 @@ cmake_minimum_required (VERSION 3.11)
 project (validator-keys-tool)
 
 #[===========================================[
-  The tool depends on the xrpl_core
+  The tool depends on the libxrpl
   library which is defined by the rippled
   project. This looks for installed rippled
   libs and, if not found, pulls them in with
   FetchContent.
 #]===========================================]
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/Builds/CMake")
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 include(KeysNIH)
```

Comment:
Do we still want this hanging around? I hope the `rippled` build isn't relying on this?

Reply:
The `rippled` build isn't relying on it. The build of this project, `validator-keys-tool`, relies on it. It is how `include(KeysNIH)` on the next line can find the `KeysNIH.cmake` module. Same for the other `include()`s.

Reply:
Whoops, what I meant was, "the `validator-keys-tool` _target_ in the rippled repo.
Like, if it's removed, we'll need to change `rippled`.

Reply:
It's two different directories. Note the `CMAKE_CURRENT_SOURCE_DIR`, which points into this project, not rippled. rippled adds its own module directory in its [listfile](https://github.com/XRPLF/rippled/blob/develop/CMakeLists.txt#L12).